### ISSUE TITLE
Missing accessibility (a11y) tests

### ISSUE THREAD

**Issue by KKoukiou** [RedHatInsights/image-builder-frontend#948]
* Easy first static check would be to add "plugin:jsx-a11y/recommended" to your eslint config. Ideally should enable it for all console.redhat.com projects

I shortly tried it out and the situation is actually really good:

```
[kkoukiou@sequioa image-builder-frontend]$ npm run lint 

> image-builder@1.1.0 lint
> npm-run-all lint:*


> image-builder@1.1.0 lint:js
> eslint config src


/home/kkoukiou/repos/image-builder-frontend/src/Components/CreateImageWizard/formComponents/ReviewStep.js
  151:11  error  The autoFocus prop should not be used, as it can reduce usability and accessibility for users              jsx-a11y/no-autofocus
  157:19  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  189:19  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  217:19  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text

/home/kkoukiou/repos/image-builder-frontend/src/Components/CreateImageWizard/formComponents/TargetEnvironment.js
  65:17  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  81:17  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  99:17  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text

✖ 7 problems (7 errors, 0 warnings)

ERROR: "lint:js" exited with 1.

```
* For more advanced tests the suggested tool to use is https://github.com/patternfly/patternfly-a11y, which is based on axe. This however implies browser based testings, which is currently missing in your tests, as you use jest tests on a fake DOM.
